server/src/main/java/com/lunchtime/controllers/FeedbackController.java

```diff
@@ -1,33 +1,27 @@
 package com.lunchtime.controllers;
 
 import java.util.List;
+
 import com.lunchtime.models.Feedback;
-import com.lunchtime.implementation.FeedbackServiceImplement;
+import com.lunchtime.service.implementation.FeedbackServiceImpl;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-//TODO remove redundant lines
-
 @RestController
 @RequestMapping("/api/feedback")
 public class FeedbackController {
+    private final FeedbackServiceImpl feedbackServiceImpl;
 
-    //TODO rename variable name to normal CamelCase
-    private final FeedbackServiceImplement feedbackServiceimplementation;
-
-    public FeedbackController(FeedbackServiceImplement feedbackServiceimplementation) {
-        this.feedbackServiceimplementation = feedbackServiceimplementation;
+    public FeedbackController(FeedbackServiceImpl feedbackServiceImpl) {
+        this.feedbackServiceImpl = feedbackServiceImpl;
     }
```

Comment:
Due to SOLID (O = "Classes should be open for extension but closed for modification"), we should use `private final FeedbackService` here, not `private final FeedbackServiceImpl`.

Morover, we can use **[@RequiredArgsConstructor](https://projectlombok.org/features/constructor)** and save code with constructor.

Reply:
Yes, I agree with SOLID, thats why few days ago I mentioned about this in one of our meetings, and You said that you have already fixed this.
PS. ..my task is just fixing TODOs, not refactoring all project, 
PS. nonetheless I can fix it too)